components/insight/SRC/org/openmicroscopy/shoola/agents/imviewer/view/ImViewerComponent.java

```diff
@@ -1082,6 +1082,33 @@ public void setLookupTable(int index, String lut, boolean preview) {
     /**
      * Implemented as specified by the {@link ImViewer} interface.
      * 
+     * @see ImViewer#setReverseIntensity(int, boolean, boolean)
+     */
+    public void setReverseIntensity(int index, boolean revInt, boolean preview) {
+        switch (model.getState()) {
+        case NEW:
+        case DISCARDED:
+            throw new IllegalStateException(
+                    "This method can't be invoked in the DISCARDED, "
+                            + "NEW or LOADING_RENDERING_CONTROL state.");
```

Comment:
That's a good point. There are actually quite a lot similar cases where the message of the exception is not really correct. Guess that just slipped in with copy/pastes. Might be worth a separate general clean-up PR. Wouldn't add it to this PR, already rather long and reviewed.